java-manta-client-unshaded/pom.xml

```diff
@@ -0,0 +1,284 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright (c) 2012-2017, Joyent, Inc. All rights reserved.
+
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at http://mozilla.org/MPL/2.0/.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>java-manta</artifactId>
+        <groupId>com.joyent.manta</groupId>
+        <version>3.1.7-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>java-manta-client-unshaded</artifactId>
+    <version>3.1.7-SNAPSHOT</version>
+    <packaging>jar</packaging>
+
+    <name>java-manta-client-unshaded</name>
+    <description>Java Manta Client</description>
+
+    <dependencies>
+        <!-- Library for authenticating requests against Manta -->
+        <dependency>
+            <groupId>com.joyent.http-signature</groupId>
+            <artifactId>apache-http-client-signature</artifactId>
+            <version>${dependency.http-client-signature.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.apache.httpcomponents</groupId>
+                    <artifactId>httpclient</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <!-- Library for making HTTP connections -->
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>${dependency.apache-http-client.version}</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>commons-logging</groupId>
```

Comment:
In the unshaded artifact, you don't want to exclude commons-logging. However, you will want to do this in the shaded module and bring in the jcl-over-slf4j instead of the common-logging dependency.